### PR TITLE
Fix everything looks wrong on step pages

### PR DESCRIPTION
**Pull request by bentshrimp**
## Describe changes

How do you think it is? Let's fix it together.

## Issue number or link
#120 
## Types of changes

What is the type of code change?
_Put an `x` in the boxes that apply_

- [x] Bugfix (changes that resolve errors)
- [ ] New feature (changes which adds functionality)
- [ ] Breaking change (big changes that affect existing functionality)
- [ ] Documentation Update

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the "README.md"
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

_Please let me know if there's anything else to explain_

### CODE REVIEW

**Comment by bentshrimp** [comment]
I was working on just step1 page, but somehow I fixed them all. haha

File: src/layout/Layout.js [hunk 1]
```diff
@@ -121,18 +127,13 @@ const StepContentsContainer = styled.div`
 
 const BottomContainer = styled.div`
   display: flex;
+  width: 100%;
```
Comment (ancy0) [inline src/layout/Layout.js:130]: 😢

File: src/components/step1/SelectContainer.js [hunk 2]
```diff
@@ -92,8 +92,9 @@ export const SelectContainer = () => {
 };
 
 const StSelectContainer = styled(Grid)`
-  width: 100%;
-  height: 100%;
+  height: 25%;
```
Comment (ancy0) [inline src/components/step1/SelectContainer.js:95]: 😢

**Review by ancy0 — COMMENTED** [review]
👍

File: src/layout/Layout.js (hunk 1 above)
Comment (bentshrimp) [reply src/layout/Layout.js:130]: got it

File: src/components/step1/SelectContainer.js (hunk 2 above)
Comment (bentshrimp) [reply src/components/step1/SelectContainer.js]: oops

**Review by ancy0 — APPROVED** [review]
😀👍

**Pull request merged**